src/plugins/jartic/datasource.py

```
@@ -0,0 +1,268 @@
+
+    async def __aexit__(self, exc_type, exc_val, exc_tb):
+        await self.downloader.close()
+        if self.cleanup_after_parse:
+            logger.info("Cleaning up cached archive files")
+            for file in self.cache_dir.glob("*.zip"):
+                try:
+                    file.unlink()
+                except Exception as e:
+                    logger.warning(f"Failed to cleanup {file}: {e}")
+
+    async def list_countries(self) -> List[Dict[str, str]]:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)